src/battle_anim_psychic.c

```diff
@@ -1023,6 +1035,53 @@ void AnimTask_SkillSwap(u8 taskId)
     task->func = AnimTask_SkillSwap_Step;
 }
 
```

Comment:
```suggestion

// [Explanation of what this task does here]
// Used by: Heart Swap
// arg 0: move target
```
Document what this new anim task does

Reply:
Alright, should be set. Also defined what the new sprite templates do